src/main/java/view/InputView.java

```diff
@@ -0,0 +1,21 @@
+package view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private final Scanner scanner = new Scanner(System.in);
+
+    public List<String> readMoveCommand() {
+        System.out.println("시작위치, 움직일기물, 목적위치를 입력해주세요(예시: 12 마 33)");
+
+        String input = scanner.nextLine();
+        if(!input.matches("\\d\\d\\s\\S\\s\\d\\d")) {
```

Comment:
```suggestion
        if (!input.matches("\\d\\d\\s\\S\\s\\d\\d")) {
```
프로젝트 전체적으로 코드 포맷팅이 적용되지 않은게 맞네요.
사소한 부분이지만 코드 포맷팅을 자주 적용하여 컨벤션을 맞춰볼까요? 🙂
협업하게 될 경우 요런 부분들로 인해 불필요한 코드 컨플릭이 발생할 수 있습니다.

[코드포맷팅 단축키 및 자동 적용](https://velog.io/@pgmjun/IntelliJ-%EC%BD%94%EB%93%9C-%EC%8A%A4%ED%83%80%EC%9D%BC%EC%9D%84-%EC%84%A4%EC%A0%95%ED%95%B4%EB%B3%B4%EC%9E%90-feat.%EC%9A%B0%ED%85%8C%EC%BD%94)

Reply:
항상 일일이 코드 포맷팅 수행하다가 깜빡하는 경우가 많았는데, 앞으로 도움이 많이 될 것 같네요. 감사합니다! 😄